### PR TITLE
Handelerror

### PR DESCRIPTION
**Pull request by amerrez**
@hamir @a27med 

"whoever changes his git habits, gets frustration.."

### CODE REVIEW

**Comment by amerrez** [comment]
y u no review :(

File: prayer_info.py [hunk 1]
```diff
@@ -13,22 +13,23 @@ def GetPrayerTimes(self, lat, lng):
...
     print '[GetPrayerTimes] salah.com scrape result = ', prayer_times
     result = {}
 
     for key in prayer_times:
       prayer = util.StringToDailyPrayer(key)
       if prayer:
         result[prayer] = prayer_times[key]
-
-    print '[GetPrayerTimes] prayer times = ', result 
+
```
Comment (hamir) [inline prayer_info.py]: can you get rid of this random whitespace?

**Comment by hamir** [comment]
Done :)

File: prayer_info.py (hunk 1 above)
Comment (amerrez) [inline prayer_info.py]: removed..

**Pull request merged**